### ISSUE TITLE
Hug with python 3.7.1 error

### ISSUE THREAD

**Issue by fordguo** [timothycrosley/hug#700]
I use hug and sqlalchemy `hug -f admin_app/app.py`,got such error:

```Unhandled exception in thread started by <function reload_checker at 0x10dd5c158>
Traceback (most recent call last):
  File "/Users/xxx/miniconda3/envs/python3/lib/python3.7/site-packages/hug/development_runner.py", line 108, in reload_checker
    if path[-4:] in ('.pyo', '.pyc'):
TypeError: 'NoneType' object is not subscriptable```

**Comment by fordguo** [comment]
#691 can fixes the problem.

**Comment by GhostofGoes** [comment]
This is fixed in the 2.4.2 release
